src/app/index.ts

```diff
@@ -1,5 +1,16 @@
 import express, { Application, Request, Response } from "express";
 import { getUsersRouter } from "../users/users.routes";
+import { getBookingsRouter } from "../bookings/bookings.routes";
+import { checkUserAuthenticated } from "../users/helpers";
+import { RawUserAccount } from "../users/users.types"
+
+declare global{
```

Comment:
I don't understand why we're doing this?

Reply:
Hope you understood why did this: to add a extra element in the request object